src/kemal/param_parser.cr

```diff
@@ -57,6 +62,23 @@ module Kemal
       end
     end
 
+    def parse_file_upload
+      HTTP::FormData.parse(@request) do |field, data, meta, headers|
+        if !meta.nil?
+          filename = meta.filename
+          if !filename.nil?
+            tempfile = Tempfile.new(filename)
+            ::File.open(tempfile.path, "w") do |file|
+              IO.copy(data, file)
+            end
+            @files[field] = FileUpload.new(tmpfile: tempfile, tmpfile_path: tempfile.path, meta: meta, headers: headers)
+          else
+            @body[field] = data.gets_to_end
```

Comment:
Yes nginx and apache do limit this already. However it's not uncommon for the framework to have a setting too. Never know if that's a benefit though. It often makes it difficult to change.  I suppose it depends if you see kemal being used without a proxy?